src/components/transactions/TransactionTemplate.vue

```diff
@@ -308,10 +308,14 @@ export default defineComponent({
 
       const { cryptoTransferDecimals, decimals } = CryptosInfo[commissionTokenLabel]
 
-      const tokenFee =
-        props.queryStatus === 'success' && typeof props.fee === 'number'
-          ? `${formatAmount(props.fee, cryptoTransferDecimals ?? decimals)} ${commissionTokenLabel}`
-          : placeholder.value
+      const isShouldShowFee =
```

Comment:
It's not gramatically correct to use "isShould...". Better alternatives:

- `shouldShowFee`
- `isFeeShown`
- `showFee`